testing/plzip/APKBUILD

```diff
@@ -1,39 +1,29 @@
 # Contributor: Klemens Nanni <kl3@posteo.org>
 # Maintainer: Klemens Nanni <kl3@posteo.org>
 pkgname=plzip
-pkgver=1.7
+pkgver=1.8
 pkgrel=0
 pkgdesc="Massively parallel lossless data compressor"
 url="http://www.nongnu.org/lzip/plzip.html"
 arch="all"
 license="GPL-2.0-or-later"
-depends=""
-depends_dev=""
 makedepends="lzlib-dev"
-install=""
 subpackages="${pkgname}-doc"
 source="https://download.savannah.gnu.org/releases/lzip/$pkgname/$pkgname-$pkgver.tar.gz"
 
-builddir="$srcdir/$pkgname-$pkgver"
 build() {
-	cd "$builddir"
 	./configure \
-		--build=$CBUILD \
-		--host=$CHOST \
```

Comment:
why removing these two?

Reply:
The script of plzip is not a GNU Autoconf configure script, it is handmade and thus doesn't recognize those variables.

same for the others removed.